gulpfile.js

```diff
@@ -949,28 +976,10 @@ gulp.task('fonttest', function () {
   return createTestSource('font');
 });
 
-gulp.task('botmakeref', function (done) {
-  console.log();
-  console.log('### Creating reference images');
+gulp.task('makeref', makeRef);
```

Comment:
Heh, this is actually how I had it before. As for gulp, I assume it uses Function.length to see how many arguments it takes.